users/ajp10304/ajp10304.h

```diff
@@ -0,0 +1,38 @@
+#include "action_layer.h"
+#include "eeconfig.h"
```

Comment:
```suggestion
```
I'm fairly certain these are already implicitly included, but test that out just to be sure.

Reply:
Yup, seems to work without them.  I've updated.